docs/pages/fase_01/metodologias.md

```diff
@@ -10,7 +10,7 @@
 
 ## Introdução
 
-Para a definição das melhores metodologias e ferramentas a serem utilizadas no projeto, primeiramente foi realizado um brainstorming com todos os membros da equipe. Além disso, houve a elaboração de um artefato para analisar e compreender as tecnologias cujos membros possuíam maior familiaridade visando adaptar o projeto à realidade do time.
+Para a definição das melhores metodologias e ferramentas a serem utilizadas no projeto, primeiramente foi realizado um brainstorming com todos os membros da equipe. Além disso, houve a elaboração de um [artefato](../pages/fase_02/iniciativasExtras/tabela_competencias?id=tabela-de-compet%c3%aancias) para analisar e compreender as tecnologias cujos membros possuíam maior familiaridade visando adaptar o projeto à realidade do time.
```

Comment:
Esse link provavelmente vai quebrar quando for pro pages, o certo seria (pages/fase_02/iniciativasExtras/tabela_competencias?id=tabela-de-compet%c3%aancias)